kubernetes/process_manifest.rb

```diff
@@ -3,6 +3,7 @@
 require "util"
 require "yaml"
 require "fileutils"
+require 'json'
```

Comment:
I try to avoid double-quotes in strings that I do not want any interpolation occurring - that was a ruby trick I was taught years ago.